### ISSUE TITLE
Build errors not reported in Error window.

### ISSUE THREAD

**Issue by gafter** [dotnet/roslyn#3676]
The following VB program is an example where a diagnostic is reported during the build.

Such diagnostics must appear in the Error window. But the error window in VS does not show the problem even after the build has failed.

```vb
Module Module1

    Sub Main()
        Dim item = Nothing
        If item Is Nothing Then
            On Error GoTo Foo
        End If
        If item IsNot Nothing Then
            Dim query = Function() item
        End If
Foo:
    End Sub

End Module
```

See also #3675

**Comment by heejaechang** [comment]
another dup of this - https://github.com/dotnet/roslyn/issues/1400
